src/views/Pools/components/PoolCard/StyledCardHeader.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { CardHeader, Heading, Text, Flex, Image } from '@pancakeswap-libs/uikit'
+import styled from 'styled-components'
+
+const Wrapper = styled(CardHeader)<{ isFinished?: boolean; activeBackground?: string }>`
+  background: ${({ isFinished, activeBackground, theme }) =>
+    isFinished ? theme.colors.backgroundDisabled : theme.colors.gradients[activeBackground]};
+`
+
+const StyledCardHeader: React.FC<{
+  earningTokenSymbol: string
+  stakingTokenSymbol: string
+  isFinished?: boolean
+}> = ({ earningTokenSymbol, stakingTokenSymbol, isFinished = false }) => {
+  const poolImageSrc = `${earningTokenSymbol}-${stakingTokenSymbol}.svg`.toLocaleLowerCase()
+  const isPromoted = earningTokenSymbol === 'CAKE'
```

Comment:
This is incorrect, if we have a pool BNB/CAKE , it will trigger this line

Reply:
Cheers Doge, have amended to `earningTokenSymbol === 'CAKE' && stakingTokenSymbol === 'CAKE'`